### PR TITLE
Fix checkBadrequest useless logic in user-permissions' graphql config

### LINKED ISSUE #5558: GraphQL login mutation doesn't throw detailed error message
**Describe the bug**
I noticed that the schema.graphql.js in the latest version of the User Permissions plugin is somewhat changed. The checkBadRequest function doesn't return an Apollo error anymore which results in a JWT error when the password and/or username is incorrect. 

**Steps to reproduce the behavior**
1. Go to config/schema.graphql.js
2. Go to line 8-14
3. Check the throw new Error 
4. This used to be a throw new ApolloError

**Expected behavior**
A clear error message when the username and/or password is incorrect when trying to log in.

**System**
- Node.js version: 13.10.1
- NPM version: 6.14.3
- Strapi version: 3.0.0-beta.19.3
- Database: MySQL / MariaDB
- Operating system: MacOS

### PR DESCRIPTION
**Pull request by Convly**
fix #5558

### CODE REVIEW

**Comment by derrickmehaffy** [comment]
This looks more like a temporary patch. @alexandrebodin wasn't there something in the works to fix logging in general?

File: packages/strapi-plugin-users-permissions/config/schema.graphql.js [hunk 1]
```diff
@@ -6,10 +6,10 @@ const _ = require('lodash');
  * @throws ApolloError if the body is a bad request
  */
 function checkBadRequest(contextBody) {
-  if (_.get(contextBody, 'output.payload.statusCode', 200) !== 200) {
-    const statusCode = _.get(contextBody, 'output.payload.statusCode', 400);
-    const message = _.get(contextBody, 'output.payload.message', 'Bad Request');
-    throw new Error(message, statusCode, _.omit(contextBody, ['output']));
+  if (_.get(contextBody, 'statusCode', 200) !== 200) {
+    const statusCode = _.get(contextBody, 'statusCode', 400);
+    const message = _.get(contextBody, 'message[0].messages[0].message', 'Bad Request');
```
Comment (alexandrebodin) [inline packages/strapi-plugin-users-permissions/config/schema.graphql.js]: I think we should change this to be contextBody.error || badRequest as it is just the type of error.

File: packages/strapi-plugin-users-permissions/config/schema.graphql.js [hunk 2]
```diff
@@ -6,10 +6,10 @@ const _ = require('lodash');
...
+    throw new Error(message, statusCode, contextBody);
```
Comment (alexandrebodin) [inline packages/strapi-plugin-users-permissions/config/schema.graphql.js]: Here the 2nd et 3rd params are not used. Can you create the Error object and set  a code property and a data property :)

**Comment by alexandrebodin** [comment]
> This looks more like a temporary patch. @alexandrebodin wasn't there something in the works to fix logging in general?

that's not about logging here

File: packages/strapi-plugin-users-permissions/config/schema.graphql.js (hunk 1 above)
Comment (Convly) [reply packages/strapi-plugin-users-permissions/config/schema.graphql.js]: Updated in f1710292431f69183bf44e7908e529814c9ea008

File: packages/strapi-plugin-users-permissions/config/schema.graphql.js (hunk 2 above)
Comment (Convly) [reply packages/strapi-plugin-users-permissions/config/schema.graphql.js]: Done in f1710292431f69183bf44e7908e529814c9ea008

**Pull request merged**